ci/azure-pipelines.yml

```diff
@@ -37,7 +37,7 @@ variables:
   - name: GOVER
     value: 1.17
   - name: NODEVER
-    value: 14.x
+    value: 18.x
```

Comment:
This one should remain at the minimum supported Node version (14.x) so the unit tests and packaging pick up accidental reliance on features that only exist in later Node versions

Reply:
I had made the changes. Please review it again.